optaplanner-integration/optaplanner-spring-boot-example/src/main/java/org/optaplanner/spring/boot/example/domain/Timeslot.java

```diff
@@ -18,37 +18,41 @@
 
 import java.time.DayOfWeek;
 import java.time.LocalTime;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 import javax.validation.constraints.NotNull;
 
-import com.fasterxml.jackson.annotation.JsonFormat;
 import org.optaplanner.core.api.domain.lookup.PlanningId;
 
+@Entity
 public class Timeslot {
 
     @PlanningId
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
     @NotNull
-    private long id;
+    private Long id;
 
     @NotNull
     private DayOfWeek dayOfWeek;
     @NotNull
-    @JsonFormat(pattern = "hh:mm")
```

Comment:
I presume these needed to be removed to get the saving of new timeslots to work?